packages/vue/src/components/molecules/SfRange/SfRange.stories.js

```diff
@@ -0,0 +1,181 @@
+import { SfRange } from "@storefront-ui/vue";
+export default {
+  title: "Components/Molecules/Range",
+  component: SfRange,
+  argTypes: {
+    value: {
+      control: {
+        type: "array",
+      },
+      table: {
+        category: "Props",
+      },
+      defaultValue: [0, 1],
+    },
+    disabled: {
+      control: "boolean",
+      table: {
+        category: "Props",
+      },
+      description: "Disabled",
+    },
+    start: {
+      control: {
+        type: "array",
+      },
+      table: {
+        category: "Props",
+      },
+      defaultValue: [0, 1],
+    },
+    range: {
+      control: "object",
+      table: {
+        category: "Props",
+      },
+      defaultValue: {
+        min: 0,
+        max: 10,
+      },
+    },
+    step: {
+      control: "number",
+      table: {
+        category: "Props",
+      },
+      defaultValue: 1,
+    },
+    connect: {
+      control: "boolean",
+      table: {
+        category: "Props",
+      },
+      defaultValue: true,
+    },
+    direction: {
+      control: "text",
+      table: {
+        category: "Props",
+      },
+      defaultValue: "ltr",
+    },
+    orientation: {
+      control: "text",
+      table: {
+        category: "Props",
+      },
+      defaultValue: "horizontal",
+    },
+    behaviour: {
+      control: "text",
+      table: {
+        category: "Props",
+      },
+      defaultValue: "tap-drag",
+    },
+    tooltips: {
+      control: "boolean",
+      table: {
+        category: "Props",
+      },
+      defaultValue: true,
+    },
+    keyboardSupport: {
+      control: "boolean",
+      table: {
+        category: "Props",
+      },
+      defaultValue: true,
+    },
+    format: {
+      control: "object",
+      table: {
+        category: "Props",
+      },
+      defaultValue: {
+        to: function (value) {
+          return new Intl.NumberFormat("de-DE", {
+            style: "currency",
+            currency: "EUR",
+          }).format(value);
+        },
+        from: function (value) {
+          const parsedValue = new Intl.NumberFormat("de-DE", {
+            style: "currency",
+            currency: "EUR",
+          }).formatToParts(value);
+          return parsedValue[0].value;
+        },
+      },
+    },
+    change: { action: "New value", table: { category: "Events" } },
+  },
+};
+
+const Template = (args, { argTypes }) => ({
+  components: { SfRange },
+  data() {
+    return {
+      rangeValue: this.value,
+    };
+  },
+  props: Object.keys(argTypes),
+  template: `
+  <div style="padding: 100px">
+    <SfRange
+      v-model="rangeValue"
```

Comment:
there is a wrong prop type in a console :)

Reply:
Thanks :)